covid_plot_cases.py

```diff
@@ -103,6 +105,50 @@ def save_cases_plots(df: pd.DataFrame) -> None:
                         + 'Proactive: Testing done at high risk locations, rather than random sampling.',
               footnote_left=f'{source}Data Source: API: Daily Reports of COVID-19 Infections')
 
+    #Thailand Covid Cases by Nationality
+    mapping = pd.DataFrame([['Thai', 'Thailand'],
```

Comment:
maybe need to comment that this won't include all possible alternative spellings and could be improved in the future?